js/core.js

```diff
@@ -1069,7 +1069,7 @@ function parsePreamble(preamble, key, value, lineno) {
 
 const importedSketches = {};
 
-async function parsePhraseBook(s, loadSketch) {
+async function parsePhraseBook(s) {
```

Comment:
Hold on — the dependency injection is still important. We want to be able to use `core.js` without `editor.js`. Removing the function passing here requires that both are loaded.